web/repos.jspf

```diff
@@ -126,7 +126,9 @@ Copyright (c) 2015, 2017, Oracle and/or its affiliates. All rights reserved.
                         </h4>
                     </div>
                         <div class="panel-body-accordion<% if (pHelper.hasFavourite(group)) { %> favourite<% } %>"
-                        <% if (pHelper.hasFavourite(group) || pHelper.getRepositories(group).size() < 5) { %>data-accordion-visible="true"<% } %>>
+                        <% if (pHelper.hasFavourite(group) ||
+                                (pHelper.getRepositories(group).size() > 0 && pHelper.getRepositories(group).size() < 5)) {
```

Comment:
not related to this however finally I see the reason for the behavior where some groups are expanded and some not. Time for a tunable ?